source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/AdaptiveCardRenderer.java

```diff
@@ -164,11 +187,22 @@ public View internalRender(RenderedAdaptiveCard renderedCard,
                 {
                     try
                     {
-                        actionLayoutRenderer.renderActions(renderedCard, context, fragmentManager, cardLayout, baseActionElementList, cardActionHandler, hostConfig, renderArgs);
+                        renderArgs.setRootLevelActions(true);
+                        View actionButtonsLayout = actionLayoutRenderer.renderActions(renderedCard, context, fragmentManager, cardLayout, primaryElementVector, cardActionHandler, hostConfig, renderArgs);
+
+                        if (!secondaryElementVector.isEmpty())
+                        {
+                            IActionLayoutRenderer overflowActionRenderer = CardRendererRegistration.getInstance().getOverflowActionLayoutRenderer();
```

Comment:
Rename `overflowActionRenderer` to `actionLayoutRenderer` to reduce confusion. 

This variable name shadows function argument `@Nullable IOverflowActionRenderer overflowActionRenderer` but it turns out not in the `IOverflowActionRenderer` type, but it's in the type of `IActionLayoutRenderer`.

Reply:
Done.